package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "amazon-connect-streams",
-  "version": "1.4.5",
+  "version": "1.4.7",
```

Comment:
already on 1.4.7 but we can do the version bump post-merge

Reply:
This was a side-effect of running `npm install` (note that this is `package-lock.json` instead of `package.json`). I can revert that change if required.